scripts/repackage_bundle.sh

```diff
@@ -42,9 +41,9 @@ do
 
     # Extra work required for the SCS plugin to support offline install.
     # We include prebuilt native code bundles for Keytar and clean up unwanted binaries.
-    if [[ $tar = *"secure-credential-store"* ]]; then
+    if [[ $tar = *"secure-credential-store"* || $tar = "zowe-cli-7"* ]]; then
```

Comment:
would it be good to add a comment as to why this apply to both , the SCS plugin as well as the CLI 7.x.x version?
it may not be obvious that CLIv7 includes keytar as an optional dependency 😋